packages/babel-preset/index.js

```diff
@@ -78,9 +75,11 @@ function preset(_, explicitOptions = {}) {
   options.targets = getTargets(env, options);
 
   if (target === 'web' || target === 'web-app') {
-    // Webpack's parser (acorn) can't object rest/spread
     options.include = [
-      ...(options.include || []),
+      ...options.include,
+      // XXX: Workaround for babel/babel#8019.
+      'transform-classes',
+      // Webpack's parser (acorn) can't handle object rest/spread
```

Comment:
@jquense acorn 5.4.0 supports object rest spread now. should we drop this?

Reply:
if webpack is up to date yeah

Reply:
AFAIK tho the problem wasn't acorn support but webpacks chosen ecma version for acorn. The comment is a bit misleading in that regard

Reply:
looks like they bumped it to 2019 in 4.12.0: https://github.com/webpack/webpack/commit/4e15f644020d0fe9a4c839504cb4c937e42b6b20